pyproject.toml

```diff
@@ -1,3 +1,6 @@
+#[project]
+#requires-python = ">=3.8"
```

Comment:
Not sure about this, this could have influenced the checks codacy and/or pylint are performing. We should double check.

But definitely needs to be updated to 3.9